manifest.php

```diff
@@ -30,7 +30,7 @@
     'label' => 'extension-tao-dac-simple',
     'description' => 'extension that allows admin to give access to some resources to other people',
     'license' => 'GPL-2.0',
-    'version' => '6.5.1',
+    'version' => '6.5.2',
```

Comment:
I'd say it's a feature, not a fix.
Although, indeed it fixes the error from popping up, it actually implements the whole new process and changes the UX.

Reply:
changed to 6.6.0